tests/lib/hqstyle-core.css

```diff
@@ -293,19 +293,13 @@ th {
 
 @font-face {
     font-family: 'Glyphicons Halflings';
-    src:
-url('https://dnwn0mt1jqwp0.cloudfront.net/static/bootstrap/fonts/glyphicons-halflings-regular.eot?f4769f9bdb74');
-    src:
-url('https://dnwn0mt1jqwp0.cloudfront.net/static/bootstrap/fonts/glyphicons-halflings-regular.eot?&f4769f9bdb74#iefix')
-format('embedded-opentype'),
-url('https://dnwn0mt1jqwp0.cloudfront.net/static/bootstrap/fonts/glyphicons-halflings-regular.woff2?448c34a56d69')
-format('woff2'),
-url('https://dnwn0mt1jqwp0.cloudfront.net/static/bootstrap/fonts/glyphicons-halflings-regular.woff?fa2772327f55')
-format('woff'),
-url('https://dnwn0mt1jqwp0.cloudfront.net/static/bootstrap/fonts/glyphicons-halflings-regular.ttf?e18bbf611f2a')
-format('truetype'),
-url('https://dnwn0mt1jqwp0.cloudfront.net/static/bootstrap/fonts/glyphicons-halflings-regular.svg?89889688147b#glyphicons_halflingsregular')
-format('svg')
+    src: url('/static/style/lib/bootstrap-3.2.0/fonts/glyphicons-halflings-regular.eot?f4769f9bdb74');
```

Comment:
It's not found in standalone. This is for bootstrap 3 icons, which aren't used in vellum - we use either fontawesome or custom HQ icons.

I could have left these links alone, but having cloudfront references seemed weird.